src/commands/configure.ts

```diff
@@ -16,7 +16,7 @@
 // ________________________________________________________________________________________________
 
 import { SlashCommandBuilder } from "@discordjs/builders";
-import { ChannelType } from "discord-api-types";
+import { ChannelType } from "discord-api-types/v9";
```

Comment:
This change should be done everywhere else we import `discord-api-types`.

Reply:
We should. I would like to do this in a follow up though

Reply:
This should be done in #94.